content/releases/cloudnative-pg-1-24.0-rc1-released/index.md

```diff
@@ -0,0 +1,167 @@
+first candidate for CloudNativePG 1.24! This release candidate showcases all
+the features expected in the final version of CloudNativePG 1.24, although some
+details may still be refined during this preview phase.
+
+For detailed information on the features and changes in CloudNativePG 1.24,
+please refer to the [release notes](https://cloudnative-pg.io/documentation/preview/release_notes/v1.24/).
+
+### Get Involved and Make a Difference!
+
+In true open source spirit, we strongly encourage you to test the new features
+of CloudNativePG 1.24 on your systems. Your testing and feedback are crucial in
+helping us identify and resolve any bugs or issues. While we do not recommend
+using CloudNativePG 1.24 RC1 in production environments, we urge you to
+simulate your typical application workloads with this preview release.
+
+Your involvement ensures that the final release of CloudNativePG 1.24 upholds
+our high standards of stability and reliability, maintaining its status as one
+of the world's most popular open source operators for PostgreSQL in Kubernetes.
+
+For more details on our preview testing process and how you can contribute,
+read more [here](https://cloudnative-pg.io/documentation/current/preview_version).
+
+---
+
+## CloudNativePG 1.24 Feature Highlights
+
+### Distributed PostgreSQL Topologies
+
+CloudNativePG 1.24 significantly enhances the replica cluster feature, enabling
+the creation of distributed database topologies for PostgreSQL across multiple
+Kubernetes clusters. This supports hybrid and multi-cloud deployments,
+offering:
+
+- **Declarative Primary Control**: Use declarative configuration to easily
+  specify which PostgreSQL cluster acts as the primary in a distributed setup.
+- **Seamless Switchover**: Demote the current primary and promote a selected
+  replica cluster, typically in a different region, without needing to rebuild
+  the former primary. This ensures high availability and resilience in diverse
+  environments.
+
+### Managed Services
+
+CloudNativePG 1.24 introduces managed services through the `managed.services`
+stanza, allowing you to:
+
+- Disable the read-only and read services via configuration.
+- Leverage the service template capability to create custom service resources,
+  including load balancers, to access PostgreSQL outside
+  Kubernetes—particularly useful for DBaaS purposes.
+
+### Enhanced API for Synchronous Replication
+
+A more powerful API has been introduced to control the configuration of
+PostgreSQL synchronous replication, supporting both quorum-based and priority
+list strategies. This update allows full customization of the
+`synchronous_standby_names` option, providing greater control and flexibility.
+
+### WAL Disk Space Exhaustion
+
+Version 1.24 includes a critical safety measure to handle WAL disk space
+exhaustion. Instead of triggering a chain of failovers when PostgreSQL runs out
+of disk space for WAL files, the cluster will safely stop. This prevents the
```

Comment:
```suggestion
exhaustion. Instead of a potential chain of failovers when PostgreSQL runs out
of disk space for WAL files, the cluster will safely stop. This prevents the
```